openpype/tools/sceneinventory/view.py

```diff
@@ -146,25 +142,25 @@ def _build_item_menu_for_selection(self, items, menu):
         switch_to_versioned = None
         if has_loaded_hero_versions:
             def _on_switch_to_versioned(items):
-                repre_ids = []
+                repre_ids = set()
                 for item in items:
-                    item_id = ObjectId(item["representation"])
+                    item_id = item["representation"]
                     if item_id not in repre_ids:
-                        repre_ids.append(item_id)
+                        repre_ids.add(item_id)
```

Comment:
Like above this could be something like:
```python
        repre_ids = {
            item["representation"]
            for item in items
        }
```

Right?